eZ/Publish/API/Repository/Values/Content/Field.php

```diff
@@ -85,4 +86,9 @@ public function getFieldTypeIdentifier(): string
     {
         return $this->fieldTypeIdentifier;
     }
+
+    public function isVirtual(): bool
```

Comment:
`@phpstan-assert-if-true` can be used to indicate a relationship between this method and others.

https://phpstan.org/writing-php-code/narrowing-types#custom-type-checking-functions-and-methods

```suggestion
    /**
     * @phpstan-assert-if-true !null $this->getId()
     */
    public function isVirtual(): bool
```